hangman.py

```diff
@@ -0,0 +1,250 @@
+# Problem Set 2, hangman.py
+# Name: Zlotenko Bohdana
+# Students group: KM-04
+# Collaborators: Alexey Garmash
+# Time spent: really a lot
+
+import random
+from string import ascii_lowercase
+
+WORDLIST_FILENAME = "words.txt"
+VOWELS = ('a', 'e', 'i', 'o', 'u')
+UNKNOWN_LETTER = "_ "
+HINT = "*"
+
+warnings = 3
+guesses = 6
+letters = ascii_lowercase
+letters_guessed = []
+word_letters_guessed = []
+
+
+def load_words():
+    """
+    Returns a list of valid words. Words are strings of lowercase letters.
+
+    Depending on the size of the word list, this function may
+    take a while to finish.
+    """
+    print("Loading word list from file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    wordlist = inFile.readline().split()
+    # wordlist: list of strings
+    print("  ", len(wordlist), "words loaded.")
+    return wordlist
+
+
+def choose_word(wordlist):
+    """
+    wordlist (list): list of words (strings)
+
+    Returns a word from wordlist at random
+    """
+    return random.choice(wordlist)
+
+
+wordlist = load_words()
+word = choose_word(wordlist)
+
+
+def get_total_score(secret_word, guesses_remaining):
+    """
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    guesses_remaining: guesses that are left
+    returns: user's score;
+
+    """
+
+    total_score = len(set(secret_word)) * guesses_remaining
+    return total_score
+
+
+def is_word_guessed(secret_word, letters_guessed):
+    """
+    secret_word: string, the word the user is guessing; assumes all letters are
+      lowercase
+    letters_guessed: list (of letters), which letters have been guessed so far;
+      assumes that all letters are lowercase
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    """
+
+    if set(secret_word) == set(letters_guessed):
+        print('-' * 20, f'\nCongratulations, you won!\nTotal score: {get_total_score(word, guesses)}')
```

Comment:
Please - try to have several prints, it's will be easier to read that (here and everywhere else)